dimod/binary/binary_quadratic_model.py

```diff
@@ -1230,9 +1263,40 @@ def from_numpy_matrix(cls, mat, variable_order=None, offset=0,
         return bqm
 
     @classmethod
-    def from_numpy_vectors(cls, linear, quadratic, offset, vartype, *,
-                           variable_order=None, dtype=np.float64):
-        """Create a binary quadratic model from NumPy vectors."""
+    def from_numpy_vectors(cls, linear: ArrayLike, quadratic: ArrayLike,
+                           offset: Number, vartype: Vartype, *,
```

Comment:
Offset's type should be `float`, [Pep484](see https://www.python.org/dev/peps/pep-0484/#the-numeric-tower)

Reply:
Fixed